### PR TITLE
Feat: potentiometer

### PR DESCRIPTION
**Pull request by imrishabh18**
Close - #171

### CODE REVIEW

File: lib/components/normal-components/Potentiometer.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      zodProps: potentiometerProps,
+      sourceFtype: "simple_potentiometer" as Ftype,
+    }
+  }
+
+  initPorts() {
+    super.initPorts({
+      additionalAliases: {
+        pin1: ["anode", "pos", "left"],
+        pin2: ["cathode", "neg", "right"],
```
Comment (seveibar) [inline lib/components/normal-components/Potentiometer.ts:31]:
```suggestion
        pin1: [ "pos", "left"],
        pin2: ["neg", "right"],
```

File: lib/components/normal-components/Potentiometer.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      },
+    })
+  }
+
+  _getSchematicSymbolDisplayValue(): string | undefined {
+    return `${formatSiUnit(this._parsedProps.maxResistance)}Ω`
```
Comment (seveibar) [inline lib/components/normal-components/Potentiometer.ts:37]: nice 👍

File: tests/components/normal-components/potentiometer.test.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  circuit.add(
+    <board width="10mm" height="10mm">
+      <potentiometer name="P1" maxResistance="10k" />
+    </board>,
+  )
+
+  circuit.render()
+
+  expect(circuit.db.schematic_component.list()).toHaveLength(1)
+  expect(circuit.db.schematic_port.list()).toHaveLength(2)
```
Comment (seveibar) [inline tests/components/normal-components/potentiometer.test.tsx:16]: can you throw in a snapshot here?

**Review by seveibar — COMMENTED** [review]
looking good! can we add a snapshot?

**Comment by imrishabh18** [comment]
@seveibar Couple of things I noticed from the snapshots are, that if the `schRotation` is 0 the component is flipped to what it is in the schematic-symbols. And when `180` rotation is also not rotating it

**Review by seveibar — APPROVED** [review]
nice find

**Pull request merged**